### ISSUE TITLE
Alert bug

### ISSUE THREAD

**Issue by adriennecoon** [ITS-UofIowa/its.uiowa.edu#77]
I received this message about a possible bug from Virginia: 

> I’ve had a problem when closing some outages that Jamie wanted me to tell you about.
> 
> I’ve been using Firefox exclusively.
> 
> These are service outages, so they are posted without an end date or time.
> 
> The service owner in several cases has been slow to tell us that something is fixed, so I’ve updated it at 3:15 or 3:30 and marked the close time as 3 or 3:15.  After I save it, the notice shows on my screen with the link to send an email update to the list, so I click the link and then nothing happens – no email notifications occur and no error is given.  Usually, you click the link, and the message is sent without any messaging or dialogue – you just wait for the message to be processed by listserv.
>
> I have only had this happen 2 – 3 times and it’s always after I have added the close date on an active outage.
>
> So these are published, and I’m going back later – either hours or days and hours to make the closing edits to the notice – editing the text of the outage to indicate it’s fixed and the adding the end date and time.

**Comment by adriennecoon** [comment]
FYI both Virginia and I have been trying to reproduce this on stage, but email isn't set up. Can you set up the alerts email to go to mine and Virginia's email address so we can test this out more? Or do you have any suggestions for why this might be occurring?

**Comment by joewhitsitt** [comment]
@adriennecoon I have enabled the email debugging on STAGE and addressed emails to go to you. I am going to let you mess around with it for a bit.

I am not sure I completely understand the steps to reproduce, but I plan to look back at the logic to see if there is anything that sticks out.

**Comment by joewhitsitt** [comment]
@adriennecoon I did a bit of digging and have some questions:

I am concerned about:
>so I click the link and then nothing happens – no email notifications occur and no error is given. Usually, you click the link, and the message is sent without any messaging or dialogue – you just wait for the message to be processed by listserv.

Because it is supposed to write an entry below the button (e.g. `Thursday, January 4, 2018 - 8:53am by {user}`). Also there should be a success message at the top of the page that says something like "Alert email sent to..."

She says she doesn't see these when this happens or ever?!

Next are there specific alert nodes where this happened that I can look at?

There could be an issue with back dating the alert and marking it close, or some combination, but it shouldn't affect the ability to send an email unless it is no longer published or the status/type were changed.

So far I am unable to reproduce in my local environment.
